File: 프로그래머스/2/12909. 올바른 괄호/올바른 괄호.js
function solution(s){
    if(s[0] == ')') return false;
    let cnt =0;
    for(let i=0; i<s.length; i++){
        s[i]=="(" ? cnt+=1 : cnt-=1;
        if(cnt<0) return false;
    }
    return cnt==0 ? true : false;
}

File: 프로그래머스/2/12909. 올바른 괄호/올바른 괄호.py
def solution(s):
    if s[0] == ')': 
        return False
    cnt=0
    for i in s:
        if i == '(':
            cnt+=1
        elif i == ')':
            cnt-=1
        if cnt<0:
            return False
    return True if cnt==0 else False
